safe/definitions/reports/infographic.py

```diff
@@ -0,0 +1,270 @@
+# coding=utf-8
+
+"""Definitions relating to infographic template elements."""
+
+from safe.utilities.i18n import tr
+from safe.utilities.resources import resources_path
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+"""Map elements"""
+
+
+map_overview = {
+    'id': 'map-overview',
+    'path': resources_path('img', 'raster_layer', 'map-overview.tif')
+}
+
+
+"""HTML frame elements"""
+
+
+people_section_notes = {
+    'id': 'people-section-notes',
+    'mode': 'text',
+    'text': '',
+    'component': 'infographic-people-section-notes'
+}
+
+population_chart_legend = {
+    'id': 'population-chart-legend',
+    'mode': 'text',
+    'text': '',
+    'component': 'population-chart-legend'
+}
+
+html_frame_elements = [people_section_notes, population_chart_legend]
+
+
+"""Header label elements"""
+
+
+infographic_header = {
+    'id': 'infographic-header',
+    'string_format': tr('Estimated impact of {impact_function_name}')
+}
+
+map_overview_header = {
+    'id': 'map-overview-header',
+    'string_format': tr('Event location')
+}
+
+population_chart_header = {
+    'id': 'population-chart-header',
+    'string_format': tr('Estimated number of people affected by hazard level')
+}
+
+people_section_header = {
+    'id': 'people-section-header',
+    'string_format': tr('People affected by hazard')
+}
+
+age_gender_section_header = {
+    'id': 'age-gender-section-header',
+    'string_format': tr('People displaced by age and gender')
+}
+
+vulnerability_section_header = {
+    'id': 'vulnerability-section-header',
+    'string_format': tr('Vulnerable people displaced by age, gender, '
+                        'and disability')
+}
+
+female_vulnerability_section_header = {
+    'id': 'female-vulnerability-section-header',
+    'string_format': tr('Displaced females by vulnerability')
+}
+
+minimum_needs_section_header = {
+    'id': 'minimum-needs-section-header',
+    'string_format': tr('Minimum needs for displaced people')
+}
+
+additional_minimum_needs_section_header = {
+    'id': 'additional-minimum-needs-section-header',
+    'string_format': tr('Additional needs for vulnerable females')
+}
+
+no_data_replacement = {
+    'id': 'no-data-replacement',
+    'string_format': tr('-')
+}
+
+minimum_needs_unit = {
+    'id': 'minimum-needs-unit',
+    'string_format': tr('{unit_abbreviation}/{frequency}')
```

Comment:
I just put it as a template, in case someday we want add some string to it. can we leave it as it is?